src/dmd/declaration.d

```diff
@@ -127,9 +129,14 @@ bool modifyFieldVar(Loc loc, Scope* sc, VarDeclaration var, Expression e1)
                         // dmd.ctorflow.CSX enum.
                         // @@@DEPRECATED_2019-01@@@.
                         if (fi & CSX.deprecate_18719)
-                            .deprecation(loc, "%s field `%s` initialized multiple times", modStr, var.toChars());
+                        {
+                            .deprecation(loc, "%s field `%s` was initialized in a previous constructor call", modStr, var.toChars());
```

Comment:
The fix for 18719 is too aggressive and only checks whether a this call is present.
Hence, there's no way to get the line number of the field with the previous initialization (there might be none after all).
See also: https://issues.dlang.org/show_bug.cgi?id=19030

Reply:
Ok. Perhaps an intermediate solution can be implemented. For example, printing the line number of the constructor call if there is no line number for the field.

Reply:
The problem is the as of now we don't have this information. It currently triggers this deprecation whenever a `this(...)` call has been observed and a immutable field is modified. This means if there are three constructors it could be any of the two other.